types/node-red-node-test-helper/index.d.ts

```diff
@@ -2,7 +2,7 @@
 // Project: https://github.com/node-red/node-red-node-test-helper#readme
 // Definitions by: Alex Kaul <https://github.com/alexk111>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// Minimum TypeScript Version: 3.1
+// Minimum TypeScript Version: 4.7
```

Comment:
why this specific version? (I could not see a TS feature being added that would require this version bump, though I could be missing this)
if anything you could folks remove this min version requirement altogether, these types are quite based on 'standard' set of TS features.

Reply:
Well, it was purely empirical.
It is related to express types which are failing with lower TypeScript.

Reply:
thanks, that legit reason completely.